skeletons/src/main/java/com/decathlon/vitamin/compose/skeletons/VitaminSkeletonColors.kt

```diff
@@ -0,0 +1,23 @@
+package com.decathlon.vitamin.compose.skeletons
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.graphics.Color
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminSkeletonColors {
+    @Composable
+    fun primary(
+        background: Color = VitaminTheme.colors.vtmnBackgroundTertiary,
+        highlight: Color = VitaminTheme.colors.vtmnBackgroundPrimary
```

Comment:
Rename `background` and `highlight` to `backgroundColor` and `highlightColor`.

Reply:
done